app/views/admins/user_entry/_close_action.haml

```diff
@@ -0,0 +1,9 @@
+- if person != @current_user.person && !person.closed_account?
+  = link_to person.owner.present? ? t("admins.user_search.close_account") : t("admins.user_search.block_account"),
+    admin_close_account_path(person.id),
+    method: :post, data: {confirm: t("admins.user_search.are_you_sure")},
+    class: "btn btn-danger btn-block"
+  = link_to person.owner.present? ? t("admins.user_search.wipe_and_close_account") : t("admins.user_search.wipe_and_block_account"),
```

Comment:
Line is too long. [132/120]

Reply:
Should be ignored, I don't like to split the ternary into multiple lines. Other Ideas?